adaptive_memory_v3.1.py

```
@@ -1397,6 +1397,38 @@ async def _get_aiohttp_session(self) -> aiohttp.ClientSession:
+        
+        Args:
+            message_content: The message content to check
+            
+        Returns:
+            bool: True if the message is a command, False otherwise
+        """
+        if not message_content:
+            return False
+        return message_content.strip().startswith("/")
+
+    def _get_last_user_message(self, messages: List[Dict[str, Any]]) -> Optional[str]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)